integrity-backend/input-chatbot-examples/generic-meta-content.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contentMetadata":
+  {
+    "name": "Chat archive bot",
+    "mime": "application/zip",
+    "description": "Chat archive bot",
+    "author":
```

Comment:
No idea where to store them, At first i thought along with the config of backend `/home/starling/.integrity/backend-config.json` but preprocess is removed from backed.

Unless you want to inject/overwrite a generic one during action_archive